### ISSUE TITLE
[Bug] Arcbound Ravager Entered With Four +1/+1 Counters

### ISSUE THREAD

**Issue by Taylor3621** [magefree/mage#9458]
Decided to try out Momir Basic for the first time, and the AI opponent pulled an [[Arcbound Ravager]]. Normally it's supposed to enter with a single +1/+1 counter on it, but something or another caused it to enter with four! While I'd be inclined to believe it felt threatened by my Serra Avenger, that's not how Magic works, so it's probably a bug. Possibly with Momir, possibly not?

![image](https://user-images.githubusercontent.com/107843203/188095791-07b872e6-2acb-4ffe-8d69-3a82114439db.png)

**Comment by jeffwadsworth** [comment]
You do realize that is a token, right?

**Comment by PurpleCrowbar** [comment]
> You do realize that is a token, right?

If they're playing [Momir basic](https://mtg.fandom.com/wiki/Momir) created creatures should be tokens, and I think it should still enter as a 1/1. The nature of the gamemode makes it hard to recreate the bug though

**Comment by Alex-Vasile** [comment]
I know nothing of this game mode, but this isn’t the first bug from it. I think this needs a custom testing class, similar to the different variations we have for commander

**Comment by Taylor3621** [comment]
> You do realize that is a token, right?

Yup, and that's the nature of Momir Basic. However, there was never anything that happened prior to make the token enter with extra counters. Best I can think of so far is that maybe it somehow got the value of the creatures on board.
